tasks/schema.js

```diff
@@ -272,7 +272,20 @@ async function migrateRelations(context) {
     )
     .flat();
 
-  await apiV9.post("/relations", [...relationsV9, ...systemFields]);
+  const mappedRelations = [...relationsV9, ...systemFields].map( realtion => {
+    return {
+      meta: realtion,
+      field: realtion.many_field,
+      collection: realtion.many_collection,
+      schema: {
+          table: realtion.many_collection,
+          column:  realtion.many_field,
+          foreign_key_table: realtion.one_collection,
+          foreign_key_column: realtion.id,
+      },
+    }
+  } );
+  await Promise.all(mappedRelations.map( async (relation) => await apiV9.post("/relations",relation)));
```

Comment:
This is going to fire all the requests at exactly the same time. We should probably do this in sequence to make sure we don't run into problems

Reply:
I did not encounter a problem, but I made it in sequence. In case of an error, it can be frustrating that the migration process stops immediately. Therefore, it may be considered to use allSettled.

Reply:
Agreed. We can still make it fail nicer though, but still do it sequentially :)